src/Microsoft.ServiceFabric.Actors/Runtime/ActorReminderState.cs

```diff
@@ -34,10 +37,26 @@ internal ActorReminderState(ActorReminderData reminder, TimeSpan currentLogicalT
             if (reminderCompletedData != null)
             {
                 this.nextDueTime = ComputeRemainingTime(currentLogicalTime, reminderCompletedData.LogicalTime, reminder.Period);
+                ActorTrace.Source.WriteInfo(

```

Comment:
Emit current logical time as well which provides insights into how next due time is calculated

Reply:
Done